src/mainboard/clevo/tgl-u/mainboard.c

```diff
@@ -30,6 +32,24 @@ static void mainboard_init(void *chip_info)
 	lpc_open_mmio_window(CONFIG_EC_CLEVO_IT5570_RAM_BASE, 0x10000);
 }
 
+static void mainboard_prune_touchpads(void *unused)
+{
+	struct device *child, *i2c_dev = pcidev_on_root(0x15, 0);
+	uint8_t data;
+
+	/*
+	 * NS5x comes with different touchpads from various vendors. Windows
+	 * gets confused when multiple touchpads are defined on the same I2C bus
+	 * in devicetree, so detect which touchpad variant is actually attached
+	 * to the mainboard and disable the other touchpads.
+	 */
+	for (child = i2c_dev->link_list->children; child; child = child->sibling)
+		if (child->path.i2c.device)
```

Comment:
```suggestion
		if (child && child->path.i2c.device)
```

Reply:
Ahh my bad, the for loop has the exit condition for child to be not null... You can revert the change

Reply:
Reverted